open_learning_ai_tutor/Intermediary.py

```diff
@@ -10,21 +10,20 @@ class GraphIntermediary2:
     def __init__(
         self,
         model,
-        assessor,
+        assessment_history=[],
         chat_history=[],
         intent_history=[],
         options=dict(),
     ) -> None:
         self.model = model
         self.options = options
-        self.assessor = assessor
         self.intent_history = intent_history
         self.chat_history = chat_history
+        self.assessment_history = assessment_history
 
     def get_prompt2(self, problem, problem_set):
-        assessment_history = self.assessor.assess(problem, problem_set)
         metadata = {}
-        assessment = assessment_history[-1].content
+        assessment = self.assessment_history[-1].content
```

Comment:
The plan is to rewrite intermediary and stratL in the next pr - this has the minimum changes to worked with the combined tutor/assesor model